src/run.html

```diff
@@ -9,11 +9,10 @@ <h2 class="text-gray-900 font-semibold text-center text-xl md:text-3xl">
       </h2>
       <span class="mt-3 max-w-md md:max-w-lg text-gray-700 text-center">
         <span data-on-run-ready="hide">
-          It’s easy to get started with Livebook, in the cloud or your local
-          machine.
+          It’s easy to get started with Livebook, on your machine or the cloud.
         </span>
         <span data-on-run-ready="show">
-          Click below to open this notebook in your configured Livebook instance
+          Click below to open this notebook in your Livebook installation
```

Comment:
This reads pretty weird to me :thinking:

```suggestion
          Click below to open this notebook in your Livebook instance
```

or we could say just "your Livebook".

Reply:
I want to avoid the word “instance” or anything that gives the connotation of server. “your Livebook” or “in a Livebook of your choice” also works. Or “in the configured Livebook”.

Reply:
"your Livebook" it is then :D